src/components/Comments/WriteComment.js

```diff
@@ -99,7 +98,7 @@ const WriteComment = ({
         {isAnonimity ? (
           <>
             {loadingUser ? (
-              <ActivityIndicator size={'small'} color={colors.bondi_blue} />
+              <ActivityIndicator size={'small'} color={COLORS.holyTosca} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been changed from `colors.bondi_blue` to `COLORS.holyTosca`. Make sure this color change aligns with your design system and provides good contrast for visibility.
```diff
- <ActivityIndicator size={'small'} color={colors.bondi_blue} />
+ <ActivityIndicator size={'small'} color={COLORS.holyTosca} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->